server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/ExportAction.java

```diff
@@ -119,10 +125,11 @@ public void handle(Request request, Response response) throws Exception {
     }
   }
 
-  private QualityProfileDto getProfile(DbSession dbSession, @Nullable String name, String language) {
-    QualityProfileDto profile = name == null ? dbClient.qualityProfileDao().selectDefaultProfile(dbSession, language)
-      : dbClient.qualityProfileDao().selectByNameAndLanguage(name, language, dbSession);
-    return checkFound(profile, "Could not find profile with name '%s' for language '%s'", name, language);
+  private QualityProfileDto getProfile(@Nullable String name, String language) {
+    try (DbSession dbSession = dbClient.openSession(false);) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this extraneous semicolon. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2959)